docker/ci/Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM pcic/pdp-base-minimal-unsafe:1.0.0
+FROM pcic/pdp-base-minimal-unsafe:pdp-python3
```

Comment:
Add a comment that this should be replaced by a release tag when that branch has been merged.